src/com/koxudaxi/ruff/Ruff.kt

```diff
@@ -59,6 +59,7 @@ val RUFF_LSP_COMMAND = when {
     else -> "ruff-lsp"
 }
 const val WSL_RUFF_LSP_COMMAND = "ruff-lsp"
+const val PY_INTERPRETER_DIRECTORY_MACRO_NAME = "PY_INTERPRETER_DIRECTORY"
```

Comment:
Should we use Screaming snake case?

Reply:
Yes: https://kotlinlang.org/docs/coding-conventions.html#property-names